frontend/src/app/shared/constants/membersRoles.ts

```diff
@@ -0,0 +1,5 @@
+export const membersRoles = {
```

Comment:
Please name the class in capital letters

Reply:
MEMBERS_ROLES ok?

Reply:
No, MembersRoles

Reply:
Fix